index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>calculator</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="screen" value="" disabled />
+    </div>
+
+    <button id="clear-button">AC</button>
+    <button class="operation-buttons">/</button>
+
+    <br />
+    <button class = "numbers-to-press">1</button>
+    <button class = "numbers-to-press">2</button>
+    <button class = "numbers-to-press">3</button>
+    <button class= "operation-buttons">*</button>
+    <br />
+    <button class = "numbers-to-press">4</button>
+    <button class = "numbers-to-press">5</button>
+    <button class = "numbers-to-press">6</button>
+    <button class= "operation-buttons">+</button>
+    <br />
+    <button class = "numbers-to-press">7</button>
+    <button class = "numbers-to-press">8</button>
+    <button class = "numbers-to-press">9</button>
+    <button class="operation-buttons">-</button>
+    <br />
+    <button class = "numbers-to-press">0</button>
+    <button id="equals-button">=</button>
```

Comment:
Whilst CSS classes generally use kebab-case, html IDs generally use camelCase, so equalsButton. This is because they are available as properties on the window object and accessing them like this `window.equalsButton` is easier than `window["equals-button"]`. You can read more about that here https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
ok